bin/test_e2e

```diff
@@ -2,6 +2,20 @@
 
 source bin/localnet/functions.sh
 
+if [ -n "$TRAVIS" ]
+then
+    echo "Travis detected, installing kernel module"
+    sudo add-apt-repository -y ppa:wireguard/wireguard && \
+    sudo apt-get update && \
+    sudo apt install -y linux-headers-$(uname -r) && \
```

Comment:
we can install it in a single line
`sudo apt-get install -y wireguard linux-headers-$(uname -r)`

Reply:
And it's better to use single too, not both `apt` and `apt-get`.